app/models/aggregated_calls_received_metric.rb

```diff
@@ -17,6 +17,10 @@ def initialize(organisation, time_period)
       end
   end
 
+  def is_applicable?
+    @totals.size.positive?
```

Comment:
`@totals.any?` perhaps?

Also, this isn't a great name either (in hindsight), maybe we should rename it? `@channels` or `@channel_totals`? Or maybe we just leave it for the great aggregation abstraction™ that's coming at some point?